site/content/entry/yuzu-progress-report-jan-2023/index.md

```diff
@@ -0,0 +1,491 @@
+So what do we do then? german77 decided that the best solution is to {{< gh-hovercard "9676" "apply a 0.99996x multiplier" >}} to the received input from the sticks, just to avoid weird games from behaving incorrectly.
+
+People have been complaining about poor Joy-Con reception since, well, ever, and honestly, there’s a limit to how much we can do, only Nintendo knows what Bluetooth black magic they use to get good range on the Switch.
+Thankfully, the new custom Joy-Con driver allows us to have more freedom on the parameters of the Bluetooth connection, and german77 discovered that {{< gh-hovercard "9683" "disabling low power mode," >}} one of the available parameters, after establishing a connection is closer to what the console does, and should help improve range to some extent.
+
+Our general recommendations still apply: if you’re using an Intel WiFI + BT combo chipset, turn off WiFi, and make sure to have your phone away from the BT chipset and the Joy-Cons.
+It’s known that off-brand generic USB dongles provide better range than WiFi + BT combo chips.
+
+Now for players interested in bending the game engines to their limit, or just wanting to have some extra fun (us boomers like to call this “cheating”), german77 has a gift for you.
+{{< gh-hovercard "9696" "Turbo button support!" >}} Also known as rapidfire, this allows to, as the name implies, repeatedly auto-press a button as fast as the game can register it.
+
+{{< imgs
+	"./turbo.png| For some reason, Megaman comes to mind"
+  >}}
+
+Enjoy finding new ways to break your games with this!
+
+## UI changes
+
+This one has been cooking for over 2 years now, sheesh.
+Morph originally started working on {{< gh-hovercard "4949" "improving the high DPI support" >}} of yuzu all the way back in November of 2020.
+Now it’s ready, and the results are great. 
+Users that run displays at DPI values over 100% can now see proper UI element scaling, particularly noticeable in the controls setting window.
+
+{{< single-title-imgs-compare
+    "Now 4K display users rejoice"
+    "./dpibug.png"
+    "./dpifix.png"
+    >}}
+
+Nothing is safe from bugs in the user interface code, and Wollnashorn found an interesting one affecting LDN.
+The Direct Connect window asks the user for an IP and nickname.
+
+{{< imgs
+	"./dc.png| We believe in recycling here, that includes pics"
+  >}}
+
+Those settings got lost every time a game was booted.
+Well, the issue was that those settings were being saved on the per-game configuration files for no reason.
+There’s no reason to include a fundamentally global setting in the per-game settings, so {{< gh-hovercard "9521" "moving them away" >}} solved the issue.
+
+byte[] solved an issue making accessing fullscreen crash, sorry, let’s say it like the bug report, making yuzu CRASH!!
+{{< gh-hovercard "9601" "Adjusting the behaviour of yuzu’s bootmanager" >}} was enough to stop the CRASH!!
```

Comment:
```suggestion
byte[] solved an issue where accessing fullscreen would cause yuzu to crash.
{{< gh-hovercard "9601" "Adjusting the behaviour of yuzu’s bootmanager" >}} was enough to stop the [CRASH!!](https://github.com/yuzu-emu/yuzu/issues/9550)
```